Doc/tutorial/datastructures.rst

```diff
@@ -679,17 +679,18 @@ intended.
 Comparing Sequences and Other Types
 ===================================
 
-Sequence objects may be compared to other objects with the same sequence type.
-The comparison uses *lexicographical* ordering: first the first two items are
-compared, and if they differ this determines the outcome of the comparison; if
-they are equal, the next two items are compared, and so on, until either
-sequence is exhausted. If two items to be compared are themselves sequences of
-the same type, the lexicographical comparison is carried out recursively.  If
-all items of two sequences compare equal, the sequences are considered equal.
-If one sequence is an initial sub-sequence of the other, the shorter sequence is
-the smaller (lesser) one.  Lexicographical ordering for strings uses the Unicode
-code point number to order individual characters.  Some examples of comparisons
-between sequences of the same type::
+Sequence objects can be compared to some objects with the same sequence type,
```

Comment:
I find this sentence confusing:

```
Sequence objects can be compared to some objects with the same sequence type,
or instance ``range()`` object is a ``sequence`` type, but they can not be
compared
```

probably you want to say:

```
Sequence objects can be compared to other objects with the same sequence type,
although there exist some exceptions to this rule like ``range()``, which is a ``sequence`` type,
but they can not be compared with other ``range()`` objects.
```

Reply:
Oh yes, that is better. Thanks!